app/templates/merchant/view_listings.html

```diff
@@ -54,12 +54,91 @@
 {% endblock %}
 
 {% block after %}
+
+    <script src="{{ url_for('static', filename='merchant.js') }}"></script>
+
     <script type="text/javascript">
-        $(document).ready(function () {
-            $('#toggle-advanced').click(function() {
-                $('.advanced-search').toggle();
+        $('#toggle-advanced').click(function() {
```

Comment:
I think you still want this in a `$(document).ready` block. It should never cause an issue since you're loading the script inline at the bottom of the document, _but_ we're not always going to be running on localhost, this could eventually be refactored into a separate file, and you're using a selector so you'd want to make sure everything is loaded.